l10n/el.js

```diff
@@ -240,6 +240,8 @@ OC.L10N.register(
     "Pager" : "Βομβητής",
     "Voice" : "Ομιλία",
     "Car" : "Αμάξι",
+    "Organization Chart": "Οργανόγραμμα",
```

Comment:
translations are handled via Transifex, not directly in our code. You can revert these changes.